src/image-copy/azext_imagecopy/create_target.py

```diff
@@ -113,11 +116,21 @@ def create_target_image(location, transient_resource_group_name, source_type, so
     else:
         snapshot_resource_group_name = transient_resource_group_name
 
+    storage_account_name = target_blob_path.split('.')[0].split('/')[-1]
+    if target_subscription:
+        subscription_id = target_subscription
+    else:
+        subscription_id = get_subscription_id(cmd.cli_ctx)
+    source_storage_account_id = resource_id(
+        subscription=subscription_id, resource_group=transient_resource_group_name,
+        namespace='Microsoft.Storage', type='storageAccounts', name=storage_account_name)
+
     cli_cmd = prepare_cli_command(['snapshot', 'create',
                                    '--resource-group', snapshot_resource_group_name,
                                    '--name', target_snapshot_name,
                                    '--location', location,
-                                   '--source', target_blob_path],
+                                   '--source', target_blob_path,
+                                   '--source-storage-account-id', source_storage_account_id],

```

Comment:
no need to specify api version? is `source-storage-account-id` a new property since specific version or always there?

Reply:
It's an existing property. It was optional formerly, but it's mandatory now.

Reply:
got it.